src/script/services/tests/security.ts

```diff
@@ -0,0 +1,16 @@
+import { SecurityDataResults } from "../../utils/interfaces";
+import { env } from "../../utils/environment";
+
+export async function testSecurity(url: string): Promise<SecurityDataResults> {
+  const encodedUrl = encodeURIComponent(url);
+  const securityUrl = `${env.testAPIUrl}/Security?site=${encodedUrl}`;
+  const fetchResult = await fetch(securityUrl);
+  if (!fetchResult.ok) {
+    throw new Error(
+      "Error fetching security report: " + fetchResult.statusText
+    );
+  }
+  const results: SecurityDataResults = await fetchResult.json();
+  console.info("Security detection completed successfully", results);
```

Comment:
Nit: app-about.ts uses single quotes, security.ts uses double quotes.

Reply:
pushed another commit that runs prettier on my latest files (including this one), looks like prettier had been turned off on my machine for a few days after I turned on settings sync in vscode.